### PR TITLE
5.3 release blog post

### PR DESCRIPTION
**Pull request by jenweber**
This PR has the boilerplate for the 5.3 release blog post. Volunteer help is needed to fill in the details. Thanks in advance! If you have push access to this repo, feel free to push to this branch as you work, or open a PR against this branch. Reviewers are also welcome to help merge code suggestions.

## Target release window
Week of Sept 18

## Volunteer writers needed

Please comment on this PR if you can help out! You can look at the latest beta and changelog for each library to see what to cover.

- [x] Write ember.js section
- [x] Write ember-cli section
- [x] Write ember-data section

## Team approvals

- [ ] ember.js
- [ ] ember-cli
- [ ] ember-data
- [ ] typescript team

### CODE REVIEW

File: content/ember-released-5-3.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+date: 2023-09-23T00:00:00.000Z
+tags:
+  - releases
+  - '2023'
+  - version-5-x
+---
+
+Today the Ember project is releasing version 5.3 of Ember.js, Ember Data, and Ember CLI. <!-- Block start: Uncomment if an LTS candidate --><!--This release of Ember.js is an LTS (Long Term Support) candidate. LTS candidates prioritize stability over the addition of new features, and have an extended support schedule.--><!-- Block end -->
+
+This release kicks off the 5.3.1 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```
Comment (kategengler) [inline content/ember-released-5-3.md]:
```suggestion
This release kicks off the 5.4 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```

File: content/ember-released-5-3.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+#### Features
+
+Ember.js 5.3 introduced 2 features.
+
+1. Feature description
+2. Feature description
+
+#### Deprecations
+
+Ember.js 5.3 introduced 0 deprecations.
```
Comment (kategengler) [inline content/ember-released-5-3.md]:
```suggestion
Ember.js 5.3 introduced 1 deprecation.
```

File: content/ember-released-5-3.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
```
Comment (kategengler) [inline content/ember-released-5-3.md:38]:
```suggestion
1. Deprecate implicit record loading in Ember Route per [RFC #774](https://rfcs.emberjs.com/id/0774-implicit-record-route-loading). Please see [the deprecation guide](https://deprecations.emberjs.com/v5.x#toc_deprecate-implicit-route-model) for more information.
```

File: content/ember-released-5-3.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+Ember.js 5.3 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug Fixes
+
+Ember.js 5.3 introduced 0 bug fixes.
+
+#### Features
+
+Ember.js 5.3 introduced 2 features.
```
Comment (kategengler) [inline content/ember-released-5-3.md]:
```suggestion
Ember.js 5.3 introduced 0 features.
```

Possibly this block and the above should be removed?

**Review by kategengler — CHANGES_REQUESTED** [review]

File: content/ember-released-5-3.md (hunk 4 above)
Comment (jenweber) [reply content/ember-released-5-3.md]: Typically I haven't deleted the 0 count sections, but that seems great to do. Removed.

**Comment by jenweber** [comment]
@gitKrystan let me know if anything should be said in this release about TypeScript. Thank you!

**Review by kellyselden — APPROVED** [review]

File: content/ember-released-5-3.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+### Changes in EmberData 5.3
+
+#### Bug Fixes
+
+EmberData 5.3 introduced 10 bug fixes. Details are in the
+[EmberData 5.3.0 release page](https://github.com/emberjs/data/releases/tag/v5.3.0).
+
+#### Features
+
+EmberData 5.3 introduced 3 features.
```
Comment (runspired) [inline content/ember-released-5-3.md]:
```suggestion
EmberData 5.3 introduced several new features.
```

Comment (runspired) [reply content/ember-released-5-3.md]: The release notes was a pretty high level overview, if we want to get into brass tacks:

- the 3 cache methods
- 5 builders each for REST/ActiveRecord/JSON:API
- 2 serialization utils for JSON:API
- 6 builder utils
- a basic lifetimes service

Depending on how you're counting thats as many as 17 new features :D

**Review by runspired — APPROVED** [review]
overall looks good, left one comment

File: content/ember-released-5-3.md (hunk 5 above)
Comment (gilest) [reply content/ember-released-5-3.md]: And a partridge in a pear tree

File: content/ember-released-5-3.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+#### Bug Fixes
+
+Ember CLI 5.3 introduced 3 bug fixes.
+
+
+#### Deprecations
+
+Ember CLI 5.3 introduced 1 deprecation.
+
+- Deprecate outputPaths build option, as output paths will no longer be predetermined under Embroider
```
Comment (bertdeblock) [inline content/ember-released-5-3.md:127]: Can we review / merge the guide as well => https://github.com/ember-learn/deprecation-app/pull/1337 ?

Comment (jaredgalanis) [reply content/ember-released-5-3.md:127]: Done ✅

File: content/ember-released-5-3.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+For more details on changes in EmberData 5.3, please review the
+[EmberData 5.3.0 release page](https://github.com/emberjs/data/releases/tag/v5.3.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
```
Comment (NullVoxPopuli) [inline content/ember-released-5-3.md:99]: in this ember-cli release, pnpm support is no longer behind a feature flag, so folks can use `--pnpm`

Comment (NullVoxPopuli) [reply content/ember-released-5-3.md:99]: (what's the proper boilerplate for new features?)

**Review by NullVoxPopuli — CHANGES_REQUESTED** [review]
Blocking on need of the addition of mentioning official pnpm support
(and maybe mentioning pnpm has worked with ember already for quite some time, but ember-cli didn't know about pnpm)

File: content/ember-released-5-3.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+
+As part of shipping serialization utils, we implemented several long-awaited cache features to support them:
+
+- `cache.changedRelationships` retrieves a diff of what has been added or removed from each relationship for a resource
+- `cache.hasChangedRelationships` answers whether any relationships have changed for a resource
+- `cache.rollbackRelationships` will reset all relationships for that resource to their "remote" state, removing any local changes.
+
+#### Deprecations
+
+EmberData 5.3 introduced 5 deprecations.
```
Comment (kategengler) [inline content/ember-released-5-3.md:77]: Can we get this merged? https://github.com/ember-learn/deprecation-app/pull/1339

Comment (jaredgalanis) [reply content/ember-released-5-3.md:77]: @kategengler Yes, I'll get it done today.

File: content/ember-released-5-3.md (hunk 7 above)
Comment (jaredgalanis) [reply content/ember-released-5-3.md:99]: @NullVoxPopuli I can add it

**Review by jaredgalanis — APPROVED** [review]

File: content/ember-released-5-3.md (hunk 8 above)
Comment (jaredgalanis) [reply content/ember-released-5-3.md:77]: ✅

File: content/ember-released-5-3.md (hunk 7 above)
Comment (jaredgalanis) [reply content/ember-released-5-3.md:99]: > (what's the proper boilerplate for new features?)

https://github.com/ember-learn/ember-blog/blob/49eea7a790ff6deba6805e5bbf88db8c4b3cb380/post-templates/emberblog-template.md

**Comment by jaredgalanis** [comment]
merging as all feedback was met

**Pull request merged**